src/app/proof-of-liabilities/get-user-liabilities.ts

```diff
@@ -0,0 +1,18 @@
+import { LedgerService } from "@services/ledger"
+import { WalletsRepository } from "@services/mongoose"
+
+export const getUserLiabilities = async () => {
+  const wallets = await WalletsRepository().listAll("BTC")
+  if (wallets instanceof Error) return wallets
+  const liabilities = wallets.map((wallet) => walletToLiability(wallet))
```

Comment:
the use of Balance keyword here is probably more relevant (versus Liabilities).

this function doesn't need to be tied to proof of reserve. we're just returning a list of balance for wallets.

this should also be paginated, probably from the client level instead of the server level.